src/common/xo.js

```diff
@@ -735,3 +742,49 @@ export const createSrLvm = (host, nameLabel, nameDescription, device) => (
 export const deleteJobsLog = id => (
   xo.call('log.delete', {namespace: 'jobs', id})
 )
+
+// Acls, users, groups ----------------------------------------------------------
+
+export const addAcl = ({subject, object, action}) => (
+  xo.call('acl.add', {subject, object, action})
+)
+
+export const removeAcl = ({subject, object, action}) => (
+  xo.call('acl.remove', {subject, object, action})
+)
+
+export const createGroup = name => (
+  xo.call('group.create', {name})
+)
+
+export const setGroupName = (id, name) => (
+  xo.call('group.set', {id, name})
+)
+
+export const deleteGroup = id => (
+  xo.call('group.delete', {id})
+)
+
+export const removeUserFromGroup = (user, group) => (
```

Comment:
In fact it will be the same for addAcl/removeAcl, there are situations where would have ro "make the effort" to get objects where we only have ids, just to pass and destructure them to get back the ids...